common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -163,6 +163,9 @@ export interface ISequencedDocumentMessage {
 
     // Timestamp when the server ticketed the message
     timestamp: number;
+
+    // Experimental field for storing the rolling hash at sequence number
```

Comment:
Can we convert this to JSDoc and add the `@alpha` tag?

[https://api-extractor.com/pages/tsdoc/doc_comment_syntax/#release-tags](https://api-extractor.com/pages/tsdoc/doc_comment_syntax/#release-tags)